Monika After Story/game/script-greetings.rpy

```diff
@@ -2615,20 +2615,83 @@ label greeting_back_from_work:
         m 1hua "Oh, welcome back, [player]!"
         menu:
             m "Did you have a good day at work today?"
+            "I had a {i}really{/i} good day today.":
+                m 1sub "That's {i}amazing{/i}, [player]!"
+                m 1hub "I'm really happy that you had such a great day today!"
+                m 1eua "I can only imagine how well you work on a good day."
+                m 1hua "Maybe you'll even move up a bit soon!"
+                m 1eua "Anyway, welcome home."
+                m 3tsbsa "Would you like your dinner, your bath, or..."
+                m 1hubfb "Ahaha~ Just kidding."
+                return
             "Yes.":
                 m 1hub "That's good!"
                 m 1eua "Remember to rest first, okay?"
-                m "That way, you'd at least have some energy before you work more on stuff."
+                m "That way, you'd have some energy before you do anything else."
                 m 1hua "But if not, you can relax with me!"
                 m 3tku "Best thing to do after a long day of work, don't you think?"
                 m 1hub "Ahaha!"
-
+                return
             "No.":
                 m 1ekc "..."
                 m 2ekc "I'd hug you right now if I were there, [player]."
-                m "I'm sorry you had a bad work day..."
+                m "I'm sorry you had a bad day at work..."
                 m 4eka "Just remember that I'm here when you need me, okay?"
-                m 1hubfa "I love you so much, [player]."
+            "I had a really bad day.":
+                m 2ekd "Aw... I'm sorry you had a bad day at work, [player]."
+                m 2ekc "I wish I could be there to give you a hug right now."
+                m 2eka "I'm just glad to be here comforting you."
+        m 2ekd "If you don't mind talking about it, what happened today?"
+        menu:
+            "I got yelled at.":
+                m 2lksdlc "Oh... That really can ruin your day."
+                m 2dsc "You're just there trying your best, and it's not good enough for someone."
+                m 2ekc "It's not always easy to forget, either."
+                m 2lksdlc "Sometimes, they're still going on about it the next day, or there'll be this bad feeling for a while."
+                m 2eka "If it's still on your mind, I think it would do you some good to relax a little."
+                m 4lksdlb "Or maybe that's what you're already doing since you're here? Ahaha~"
+                m 1hua "I'm sure you'll feel better after we spend some together."
+            "I got passed over for someone else.":
+                m 1lksdla "Yeah... It can really ruin your day to see someone else get recognition."
```

Comment:
```suggestion
                m 1lksdld "Oh... {w=0.5}It can really ruin your day to see someone else get the recognition that you thought you deserved."
```